### ISSUE TITLE
Add tests for the backend

### ISSUE THREAD

**Issue by annikakrause** [amosproj/amos2022ss04-digital-identity#119]
## User story

1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria

- add tests to backend
- assure that no tests are failing
- Services:
  - [ ] AuthenticationService
  - [ ] CredentialDefinitionService
  - [ ] DIConnectionService
    - [x] DIConnectionService.create
  - [ ] HttpService
  - [ ] LissiApiService
  - [ ] SchemaService
  - [x] StrongPasswordService
## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release
